e2e/src/usecases/FiatConnectTransferOut.js

```diff
@@ -0,0 +1,105 @@
+import { reloadReactNative } from '../utils/retries'
+import { enterPinUiIfNecessary, quickOnboarding, waitForElementId } from '../utils/utils'
+import { ALFAJORES_FORNO_URL, SAMPLE_PRIVATE_KEY } from '../utils/consts'
+import { newKit } from '@celo/contractkit'
+import { generateKeys, generateMnemonic } from '@celo/utils/lib/account'
+
+/**
+ * From the home screen, navigate to the FiatExchange screen (add/withdraw)
+ *
+ * @return {{result: Error}}
+ */
+async function navigateToFiatExchangeScreen() {
+  await reloadReactNative()
+  await waitForElementId('Hamburger')
+  await element(by.id('Hamburger')).tap()
+  await element(by.id('add-and-withdraw')).tap()
+}
+
+/**
+ * Fund a wallet, using some existing wallet.
+ *
+ * @param senderPrivateKey: private key for wallet with funds
+ * @param recipientAddress: wallet to receive funds
+ * @param stableToken: ContractKit-recognized stable token
+ * @param amountEther: amount in "ethers" (as opposed to wei)
+ */
+async function fundWallet(senderPrivateKey, recipientAddress, stableToken, amountEther) {
+  const kit = newKit(ALFAJORES_FORNO_URL)
+  const { address: senderAddress } = kit.web3.eth.accounts.privateKeyToAccount(senderPrivateKey)
+  kit.connection.addAccount(senderPrivateKey)
+  const tokenContract = await kit.contracts.getStableToken(stableToken)
+  const amountWei = kit.web3.utils.toWei(amountEther, 'ether')
+  await tokenContract.transfer(recipientAddress, amountWei.toString()).send({ from: senderAddress })
+}
+
+export const fiatConnectNonKycTransferOut = () => {
+  it('First time FiatConnect cash out', async () => {
+    const cashOutAmount = 0.02
+    const amountPlusGas = `${cashOutAmount + 0.005}`
+    const token = 'cUSD'
+    const mnemonic = await generateMnemonic()
+    const { address: walletAddress } = await generateKeys(mnemonic)
+    await quickOnboarding(mnemonic) // ends on home screen
+    await fundWallet(SAMPLE_PRIVATE_KEY, walletAddress, token, amountPlusGas)
```

Comment:
`SAMPLE_PRIVATE_KEY` corresponds to the wallet address listed here right? https://github.com/valora-inc/wallet/blob/2d7ce14c9292bc78287f1db45dd5965fb3241a24/e2e/scripts/consts.ts#L1-L2
Is the intent that we fund the new address using this address and then the periodic job funds the wallet address from the faucet? Any reason to do this instead of directly using wallet address for tests?

Reply:
@satish-ravi the mnemonic and private key for the e2e test faucet secret. This keeps someone from draining our e2e test faucet quickly. It looks like the `SAMPLE_PRIVATE_KEY` corresponds to the `E2E_TEST_WALLET`.

Reply:
Flow of funds: `E2E_TEST_FAUCET` -> `E2E_TEST_WALLET` aka `SAMPLE_WALLET` via a script, then for each of these tests, `SAMPLE_WALLET` -> `new walletAddress` via my new helper function `fundWallet`.

Reply:
I didn't investigate why, since that actually seemed like desirable behavior-- and even if it was a fluke from not reinstalling, we might want to build it out more fully in the future. Moreover, if we made the mock provider follow the spec more closely, we could run into issues like a [ResourceExists error](https://github.com/fiatconnect/specification/blob/main/fiatconnect-api.md#3431313-resourceexists) from submitting the same fiat account info multiple times for the same address. 

Testing first-time user flows with a brand new wallet each time seemed cleaner.